src/EPR.Payment.Service.Data.UnitTests/Repositories/PaymentsRepositoryTests.cs

```diff
@@ -0,0 +1,199 @@
+﻿using AutoFixture.MSTest;
+using EPR.Payment.Service.Common.Data;
+using EPR.Payment.Service.Common.Data.Repositories;
+using EPR.Payment.Service.Common.UnitTests.Mocks;
+using EPR.Payment.Service.Common.UnitTests.TestHelpers;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using Moq.EntityFrameworkCore;
+using System.Data.Entity;
+
+namespace EPR.Payment.Service.Data.UnitTests.Repositories
+{
+    [TestClass]
+    public class PaymentsRepositoryTests
+    {
+        private Mock<DbSet<Common.Data.DataModels.Payment>> _paymentMock;
+        private Mock<DbSet<Common.Data.DataModels.Lookups.PaymentStatus>> _paymentStatusMock;
```

Comment:
Initialize with not null and remove the unnecessary namespace;
        private Mock<DbSet<Common.Data.DataModels.Payment>> _paymentMock = null!;
        private Mock<DbSet<PaymentStatus>> _paymentStatusMock = null!;

Reply:
resolved